tests/toolchain/crash.pm

```diff
@@ -17,49 +17,25 @@ use utils;
 
 sub install_kernel_debuginfo {
     # kernel debug symbols are huge, this can take a while
-    assert_script_run 'zypper ref; zypper -n -v in kernel-default-base-debuginfo kernel-default-debuginfo', 1200;
+    script_run 'rpmquery -a kernel-default*';
```

Comment:
why is this command allowed to fail and how do you use it? is it just for debugging? It should help to also call this one with `assert_script_run` because if it fails, something is clearly wrong, isn't it?